packages/flutter/lib/src/material/carousel.dart

```diff
@@ -15,31 +16,60 @@ import 'theme.dart';
 
 /// A Material Design carousel widget.
 ///
-/// The [CarouselView] present a scrollable list of items, each of which can dynamically
+/// The [CarouselView] presents a scrollable list of items, each of which can dynamically
 /// change size based on the chosen layout.
 ///
+/// Material Design 3 introduces 4 carousel layouts:
```

Comment:
Are all of these layouts supported by this widget? Maybe clarify in the docs what is supported right here.

Reply:
Updated! Thanks!